app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,14 @@
+class AccountActivationsController < ApplicationController
+  def edit
+    user = User.find_by email: params[:email]
```

Comment:
If you don't find the user with the params[email] ??
You can do the same thing what you have done in the chapter 10 using filter.